Submodels/Parameterizations.py

```
@@ -0,0 +1,775 @@
+
+        Returns
+        -------
+        None
+        """
+        
+        return
+
+
+    def BezierCurve3(self,
+                     coeff: list[float], 
+                     u: float
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)